src/components/UserDetail.tsx

```diff
@@ -6,10 +6,12 @@ import Loading from './Loading';
 import Stars from './Stars';
 import PohModule from '../modules/Poh/PohModule';
 import useUserById from '../hooks/useUserById';
+import usePaymentsForUser from '../hooks/usePaymentsForUser';
 
 function UserDetail({ user }: { user: IUser }) {
   const { user: currentUser } = useContext(TalentLayerContext);
   const userDescription = user?.id ? useUserById(user?.id)?.description : null;
+  const payments = usePaymentsForUser(user.id || '1');
```

Comment:
two problems with this logic: 
- we query all payments for one user even it's not the connected one 
- we just do it for the length

todo: 
- remove it, always display the link and just add a message on the final page if the there is no payment yet